File: tests/fixtures/java_project/src/main/java/com/example/Inventory.java
package com.example;

/** Inventory module — fixture for tests. */
public class Inventory {

    private Pricing pricing = new Pricing();

    /** Check stock and return price. */
    public double checkStock(int itemId, int qty) {
        return pricing.calculatePrice(itemId, qty);
    }

    /** Reserve items in inventory. */
    public boolean reserveItems(int itemId, int qty) {
        return qty > 0;
    }
}

File: tests/fixtures/csharp_project/Inventory.cs
using System;
using Example;

namespace Example
{
    /// <summary>Inventory module — fixture for tests.</summary>
    public class Inventory
    {
        private Pricing _pricing = new Pricing();

        /// <summary>Check stock and return price.</summary>
        public double CheckStock(int itemId, int qty)
        {
            return _pricing.CalculatePrice(itemId, qty);
        }

        /// <summary>Reserve items in inventory.</summary>
        public bool ReserveItems(int itemId, int qty)
        {
            return qty > 0;
        }
    }
}
